src/test/run-make/incr-prev-body-beyond-eof/Makefile

```diff
@@ -0,0 +1,16 @@
+-include ../../run-make-fulldeps/tools.mk
+
+# Tests that we don't ICE during incremental compilation after modifying a
+# function span such that its previous end line exceeds the number of lines
+# in the new file, but its start line/column and length remain the same.
+
+SRC=$(TMPDIR)/src
+INCR=$(TMPDIR)/incr
```

Comment:
I'm not that familiar with the run-make harness -- @Mark-Simulacrum do you know by any chance if TMPDIR is specific to this *test*?